numba/runtime/nrt.c

```diff
@@ -233,6 +247,18 @@ NRT_MemInfo *NRT_MemInfo_alloc_safe(size_t size) {
     return mi;
 }
 
+NRT_MemInfo* NRT_MemInfo_alloc_dtor_safe(size_t size, NRT_dtor_function dtor) {
```

Comment:
This function can probably be unified with `NRT_MemInfo_alloc_safe`, no? It seems the `(void *) size` above should not be useful anymore...